src/shared/src/components/organisms/AltinnReceipt.tsx

```diff
@@ -141,22 +141,27 @@ export function ReceiptComponent(props: IReceiptComponentProps) {
         >
           {props.body}
         </Typography>
-        {props.titleSubmitted && (
-          <Typography
-            variant='h3'
-            style={{
-              paddingTop: '4.1rem',
-              paddingBottom: '0.5rem',
-              fontWeight: 600,
-            }}
-          >
-            {props.titleSubmitted}
-          </Typography>
-        )}
-        <AltinnAttachmentComponent
-          attachments={props.pdf}
-          id='attachment-list-pdf'
-        />
+        {props.pdf && props.pdf.length
+          ? <React.Fragment>
+              {props.titleSubmitted && (
+              <Typography
+                variant='h3'
+                style={{
+                  paddingTop: '4.1rem',
+                  paddingBottom: '0.5rem',
+                  fontWeight: 600,
+                }}
+              >
+                {props.titleSubmitted}
+              </Typography>
+            )}
+            <AltinnAttachmentComponent
+              attachments={props.pdf}
+              id='attachment-list-pdf'
+            />
+          </React.Fragment>
+          : null
+        }
```

Comment:
nitpick; could keep using the `&&` here, to avoid the redundant `null` render. 

Not hugely important right now, but these inline conditionals tend to get quite complex over time and sometimes difficult to read. Could extract the conditional statement for easier readability, and/or change to optional chaining. 

```jsx

{props.pdf?.length && (
//...


// or before return
const hasAttachments= props.pdf?.length;

{hasAttachments && (
//...

```

```suggestion
        {props.pdf && props.pdf.length && (
          <React.Fragment>
              {props.titleSubmitted && (
              <Typography
                variant='h3'
                style={{
                  paddingTop: '4.1rem',
                  paddingBottom: '0.5rem',
                  fontWeight: 600,
                }}
              >
                {props.titleSubmitted}
              </Typography>
            )}
            <AltinnAttachmentComponent
              attachments={props.pdf}
              id='attachment-list-pdf'
            />
          </React.Fragment>
          )
        }
```

Reply:
Nice! :1st_place_medal: Not used to doing it that way, but I'll make a note of this!

Reply:
Double nitpick:
`{array.length && renderContent()}` and `{array.length ? renderContent() : null}` is slightly different in that the first will render `0` when the array is empty, but the second will render nothing. You need to write `{array.length>0 && renderContent()}` in order for the first part of the `&&` expression to be false.

Reply:
Fantastic catch, @ivarne! Fixed now.

Reply:
I used to write `&&`, but have since been convinced to do `{cond?render:null}` instead, probably because of this issue.

Reply:
Well, thats a bit surprising to me, but now I know. Thanks @ivarne 😁